### ISSUE TITLE
3d and a11y modules conflict

### ISSUE THREAD

**Issue by christopher-szu** [highcharts/highcharts#6885]
#### Expected behaviour
no js error

#### Actual behaviour
including both 3d and a11y modules at the same time will cause the series init wrapper function in a11y module to be skipped, thus skipping the creation of `types` property in `chart` object altogether.
https://github.com/highcharts/highcharts/blob/v5.0.12/js/modules/accessibility.src.js#L242-L268

error tracktrace
```accessibility.src.js:188 Uncaught TypeError: Cannot read property 'length' of undefined
    at object.H.Series.buildSeriesInfoString (http://code.highcharts.com/5.0.12/modules/accessibility.src.js:188:34)
    at object.H.Series.setA11yDescription (http://code.highcharts.com/5.0.12/modules/accessibility.src.js:178:30)
    at object.<anonymous> (http://code.highcharts.com/5.0.12/modules/accessibility.src.js:139:22)
    at object.obj.(anonymous function) [as render] (https://code.highcharts.com/5.0.12/highcharts.src.js:922:28)
    at https://code.highcharts.com/5.0.12/highcharts.src.js:16907:27
    at Array.forEach (<anonymous>)
    at H.each (https://code.highcharts.com/5.0.12/highcharts.src.js:1687:44)
    at H.Chart.renderSeries (https://code.highcharts.com/5.0.12/highcharts.src.js:16905:17)
    at H.Chart.<anonymous> (https://code.highcharts.com/5.0.12/highcharts-3d.src.js:1276:25)
    at H.Chart.obj.(anonymous function) [as renderSeries] (https://code.highcharts.com/5.0.12/highcharts.src.js:922:28)
```

#### Live demo with steps to reproduce
using the scatter plot demo, but added 3d and a11y modules.

http://jsfiddle.net/zp3ok2n8/2/

#### Affected browser(s)
only tested in chrome 59, but should affect all browsers.

**Comment by sebastianbochan** [comment]
Thank you for the reporting.

_Internal note_
@oysteinmoseng the `this.chart.types` is undefined, as a result length cannot be called.

**Comment by oysteinmoseng** [comment]
Should be fixed in current version: http://jsfiddle.net/zp3ok2n8/4/
